src/topic/BecomeMemberContainer.js

```diff
@@ -17,21 +20,36 @@ class BecomeMemberContainer extends React.Component {
         attend(topic, user);
     };
 
+    enableTooltipForAnonymous = (anonymous, component) => {
+        return anonymous ? withAuthPopover(component) : component;
+    };
+
     render() {
         const { topic, user } = this.props;
+        const anonymous = !user;
```

Comment:
functions are usually a function indicator for a sake of readability (code like a sentence): ` if ( user.isLogged() )`  and clear indication, which variable is used for such.

I mostly try to leverage it from Java convention, as JS specs (like airbnb) are not so strict about the naming - and for me it makes sense, as we can distinguish between function and var.

```
8.3.2 Boolean properties

In addition, for boolean properties, we allow a getter method to match the pattern:
public boolean is<PropertyName>();

This “is<PropertyName>” method may be provided instead of a “get<PropertyName>” method, or it may be provided in addition to a “get<PropertyName>” method. In either case, if the “is<PropertyName>” method is present for a boolean property then we will use the “is<PropertyName>” method to read the property value. An example boolean property might be:

public boolean isMarsupial();
public void setMarsupial(boolean m);
```

Feel free to shoot a few points why it should be `isAnonymousUser`, thanks in advance!

Reply:
What? Java convention? Nooo... ;-)

Reply:
Ok, but only a few:
1) just because `anonymous = !user;` is not true, because anonymus is a user, although maybe not of that particular class
2) anonymus is a boolean and this fact could be told to the programmer by good naming